test/h5test.c

```diff
@@ -2212,10 +2480,10 @@ h5_using_default_driver(const char *drv_name)
     assert(H5_DEFAULT_VFD == H5FD_SEC2);
 
     if (!drv_name)
-        drv_name = getenv(HDF5_DRIVER);
+        drv_name = h5_get_test_driver_name();
 
     if (drv_name)
-        return (!strcmp(drv_name, "sec2") || !strcmp(drv_name, "nomatch"));
+        return !strcmp(drv_name, "sec2");
```

Comment:
It would be nice if this used the same symbol as the library to determine the default VFD, instead of hard-coding the "sec2" string here.

Reply:
Seems easy enough to add a `H5_DEFAULT_VFD_NAME` macro if that's what you mean

Reply:
Added this and updated a few occurrences that seemed reasonable to change.